packages/integration_test/android/build.gradle

```diff
@@ -26,7 +26,7 @@ rootProject.allprojects {
 apply plugin: 'com.android.library'
 
 android {
```

Comment:
You've dropped the `namespace`; without that people still can't use AGP 8.0 in any project that depends on `integration_test`. That's the one change we absolutely *do* need to make for compatibility, based on my testing.

Reply:
fixed